parrot/test/TR_parrot_bind.sh

```diff
@@ -82,14 +86,14 @@ EOF
 run()
 {
 	set -e
-	parrot -- "$(pwd)/$exe"
-	parrot --work-dir=/http -- "$(pwd)/$exe" && return 1
+	parrot -- "$(pwd)/$exe" /tmp/foo
+	parrot -- "$(pwd)/$exe" /http/foo && return 1
```

Comment:
I think it's making sure that non-writable directories like `/http` don't allow sockets. That seemed a little odd, but it's how the test was written. Do you want to get rid of that?